### PR TITLE
Ignore case when parsing LogLevel string in the Console Logger Settings.

### PR DESCRIPTION
**Pull request by Yves57**

### CODE REVIEW

**Comment by Yves57** [comment]
See #381

**Comment by cesarbs** [comment]
I like it. cc @BrennanConroy @kichalla

**Comment by BrennanConroy** [comment]
Yeah, it looks good

**Comment by BrennanConroy** [comment]
Although looks like some tests are failing

File: src/Microsoft.Extensions.Logging.Console/ConfigurationConsoleLoggerSettings.cs [hunk 1]
```diff
@@ -62,7 +62,7 @@ public bool TryGetSwitch(string name, out LogLevel level)
                 level = LogLevel.None;
                 return false;
             }
-            else if (Enum.TryParse<LogLevel>(value, out level))
+            else if (Enum.TryParse<LogLevel>(value, true, out level))
```
Comment (pranavkm) [inline src/Microsoft.Extensions.Logging.Console/ConfigurationConsoleLoggerSettings.cs]: Could you use a named param here - `Enum.TryParse<LogLevel>(value, ignoreCase: true, result: out level)`?

**Comment by pranavkm** [comment]
@BrennanConroy those tests seem pretty unrelated.

**Comment by moozzyk** [comment]
I am not sure if it adds any value. You can make typos everywhere... The exception message when there is a mismatch could be better though - it could list all the possible values.

**Comment by hishamco** [comment]
This is a duplicate of PR #401

**Comment by muratg** [comment]
Dup of #401

**Pull request closed without merging**